reana_db/models.py

```diff
@@ -154,6 +158,8 @@ def __init__(self,
             self.run_number = 1
         else:
             self.run_number = last_workflow.run_number + 1
+        self.workspace_path = workspace_path or self.get_workspace()
```

Comment:
Only if no one else is calling `get_workspace` anymore, we could just call `build_workspace_path` directly and get rid of `get_workspace` method:
```suggestion
        self.workspace_path = workspace_path or build_workspace_path(owner_id, id_)

```

I think it would be more clear, read as "if there is no workspace path, one will be built", WDYT?

Reply:
Can't we apply the previous comment diff, just as it is? That would allow us to completely get rid of `get_workspace`, I don't see it being used anywhere:
```console
$ rg "get_workspace\(\)"
reana-db/reana_db/models.py
161:        self.workspace_path = workspace_path or self.get_workspace()
```